apitest/scripts/version-parser.bash

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# Bats helper script for parsing current version from Version.java.
+
+export CURRENT_VERSION=$(grep "String VERSION =" common/src/main/java/bisq/common/app/Version.java | sed 's/[^0-9.]*//g')
```

Comment:
Nit: wouldn't it be better to script this in gradle instead of bash?

It would be platform independent and it could be tested by more devs (not restricted to those using mac or linux).

It would also make integration with the current codebase easier, like accessing `Version.CURRENT_VERSION` natively here.

Reply:
This script is loaded by a couple of api test cases in the bash based [bats](https://github.com/bats-core/bats-core) test script (`mainnet-test.sh`), which I'm pretty sure does not integrate with gradle or java.   This is an ugly solution, but I don't think running a gradle or java script could be any prettier.

Also, `apitest`,  and the API daemon & cli were not intended to run on Windows, at least in the short & medium term.

Reply:
I see, ok, wasn't aware of that.